release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -24,16 +24,13 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 You must use {op-system} machines for the control plane, and you can use either {op-system} or {op-system-base} for compute machines.
 //Removed the note per https://issues.redhat.com/browse/GRPA-3517
 
-For {product-title} 4.12 on `x86_64` architecture, Red Hat has added a 6-month Extended Update Support (EUS) phase that extends the total available lifecycle from 18 months to 24 months. For {product-title} 4.12 running on 64-bit ARM (`aarch64`), {ibm-power-name} (`ppc64le`), and {ibm-z-name} (`s390x`) architectures, the EUS lifecycle remains at 18 months.
+//Even-numbered release lifecycle verbiage
+For {product-title} {product-version}, the total available lifecycle is 24 months on all supported architectures, including `x86_64`, 64-bit ARM (`aarch64`), {ibm-power-name} (`ppc64le`), and {ibm-z-name} (`s390x`) architectures, due to the Extended Update Support (EUS) phase for even-numbered releases. This 24-month lifecycle first applied to {product-title} version 4.14.
```

Comment:
> This 24-month lifecycle first applied to {product-title} version 4.14.
Per https://access.redhat.com/support/policy/updates/openshift#maintenancesupport, seems like the the 24-month lifecycle commences from 4.12, but applied to all arches from 4.14.